docs/google_react_native_signing.md

```diff
@@ -1 +1,147 @@
-# google react native signing
\ No newline at end of file
+# Google signing for React Native apps
+
+
+Refer to React Native's [Publishing to Google Play Store](https://reactnative.dev/docs/signed-apk-android) guide for an overview of app signing. That guide describes how to sign an app from a developer's computer. This page describes how to setup those steps in a CI/CD pipeline using GitHub Actions.
+
+## Prerequisite
```

Comment:
It would be nice to see this page restructured to use steps/numbers to walk someone through what they need to do.

Reply:
I added the steps at the top of the page, with links to the sections. I wasn't sure how to restructure it into steps, as each step had a fair bit of detail.